tools/operator/launcher/k8s/health.go

```diff
@@ -29,7 +30,7 @@ func (k K8s) VerifyContainersAreRunning() error {
 				if status == "No resources found." {
 					return nil
 				}
-				k.Arguments = []string{"get", "pods", "--template", `{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}`, "--field-selector=status.phase!=Running"}
+				k.Arguments = []string{"-n", ns, "get", "pods", "--template", `{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}`, "--field-selector=status.phase!=Running"}
```

Comment:
Replace with a call to the SDK instead, we want to kill of using the CLI completely.

Reply:
yes replaced with Kubernetes sdk call